generate_R_table.py

```diff
@@ -1,3 +1,4 @@
+#!/Users/antonmalko/anaconda/bin/ python
```

Comment:
Oh wait, I was thinking about another script... We can delete that line altogether; I think, I put it there to debug something and then forgot to remove